### ISSUE TITLE
Opentoonz crashes whenever I click on the raster brushes option

### ISSUE THREAD

**Issue by SakiHari** [opentoonz/opentoonz#2630]
**Issue Summary**
I just downloaded Opentoonz on my computer, when I attempt to use any of the raster brushes the program just shuts down. I don't even get the chance to click on or view any of the raster brushes, the program just crashes as soon as I click on the raster tab; I can click and use everything else except raster. 

I don't know what changed, but when I had first installed the program I could actually view and click on the brushes. I closed the program for a little while to finish some school work and when I opened it again this problem started happening.

**Comment by RodneyBaker** [comment]
I just ran into this myself and someone pointed me to the solution.
For my part I tried to remove all instances of the MyPaint program and still I'd get the crash when touching the Raster tab in the Style Editor.  The fix (for me) was to open my user directory (C:\Users\<my name>) go to the Apps directory and remove the MyPaint folder.  After that the problem immediately went away.

I have not explored the actual cause of the problem but as others have speculated it is likely that one of the new Mypaint brushes that is incompatible with Opentoonz is the problem.  Just removing brushes from OT's stuff/library folder isn't enough (although if a nerfarious brush type is in there it will cause problems).  Removing the Mypaint data in the Apps directory should do the job.

Give that a try and see if it helps.

It's odd that this suddenly showed up.
I've had MyPaint installed on this system for about a year with no issues.
